tests/python/pants_test/backend/python/tasks/native/test_ctypes_integration.py

```diff
@@ -33,6 +35,10 @@ def wrapper(*args, **kwargs):
   return wrapper
 
 
+_IS_OSX = sys.platform.startswith('darwin')
```

Comment:
It’s possible to do:
```python
from pants.backend.native.config.environment import Platform

_IS_OSX = Platform.current == Platform.darwin
```

Reply:
Done. I don't like it though - `Platform.darwin` is way too magic. Its completely unclear without reading alot of code that the member even exists.